training/model_management/src/azureml/model/mgmt/processors/hftransformers/vision/requirements.txt

```diff
@@ -0,0 +1,3 @@
+--extra-index-url https://automlcesdkdataresources.blob.core.windows.net/finetuning-models/eval_metrics_whls/17_03_latest_eval_wheel/azureml_evaluate_mlflow-0.1.0.86769376-py3-none-any.whl
```

Comment:
This should point to PyPI instead.

Reply:
@ShipraJain01 @skasturi for above comment

Reply:
Done. Changed to use the PyPi one.